### PR TITLE
feat: address books

### PR DESCRIPTION
**Pull request by simboonlong**
### pr
- add thirdparty api to be searchable in addressbook

### todo
- [x] style dropdown
- [x] add debounce
- [x] write tests

### user story
- https://www.pivotaltracker.com/story/show/174667717

### preview
Screenshot 2020-10-08 at 2 17 25 PM

### CODE REVIEW

File: src/components/UI/Overlay/OverlayContent/AddressBook.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const AddressBook = styled(({ onAddressSelected, ...props }: AddressBookP
   const [addressBookThirdPartyResults, setAddressBookThirdPartyResults] = useState<AddressBookThirdPartyResultsProps[]>(
     []
   );
-  const { name, endpoint, apiHeader, apiKey } = thirdPartyAPIEndpoints[remoteEndpointIndex];
+  const { name, endpoint, apiHeader, apiKey } = thirdPartyAPIEndpoints[remoteEndpointIndex] ?? [];
```
Comment (yehjxraymond) [inline src/components/UI/Overlay/OverlayContent/AddressBook.tsx]: is the ?? [] even the right type? shouldnt it be `?? {}`?

Comment (simboonlong) [reply src/components/UI/Overlay/OverlayContent/AddressBook.tsx]: thirdparty 1 is [], local one is {}
 
Screenshot 2020-10-09 at 5 11 51 PM

Comment (simboonlong) [reply src/components/UI/Overlay/OverlayContent/AddressBook.tsx]: oops yes right

**Review by yehjxraymond — APPROVED** [review]
lgtm

**Pull request merged**